.yamato/upm-ci-universal_stereo.yml

```diff
@@ -21,8 +21,8 @@ platforms:
 testplatforms:
   - name: Standalone
     args: --suite=playmode --platform=Standalone
-  - name: playmode
-    args: --suite=playmode
+  # - name: playmode
+  #   args: --suite=playmode
   - name: editmode
```

Comment:
What's the issue with disabling playmode stereo tests? Playmode tests are important too.

Reply:
@DennisDeRykeUnity is looking into it. There's an issue with the gameview resize.
Note that with this PR, we do add ALL the URP existing tests in playmode mode. So in the end, we have more coverage than before.